Discovery/Discovery/Presentation/WebPrograms/ProgramWebviewViewModel.swift

```diff
@@ -14,6 +14,9 @@ public class ProgramWebviewViewModel: ObservableObject {
     @Published var courseDetails: CourseDetails?
     @Published private(set) var showProgress = false
     @Published var showError: Bool = false
+    @Published var updatingCookies: Bool = false
+    private var retryCount = 1
+    
     var errorMessage: String? {
         didSet {
             withAnimation {
```

Comment:
it's bad practice to use `withAnimation` block inside view model, better to use animation modifier in that case, something like `.animation(.linear(duration: 0.25), value: showError)`

Reply:
This is what we are using almost everywhere in the app. We might create a task to update it for the whole app. What do you say?

Reply:
@sergeymomot can you create ticket about that? But as we are here, @saeedbashir could you change it in that view?

Reply:
> @sergeymomot can you create ticket about that? But as we are here, @saeedbashir could you change it in that view?

https://github.com/openedx/openedx-app-ios/issues/277